crates/shared/src/sources/balancer_v2/pool_fetching.rs

```diff
@@ -370,6 +384,54 @@ mod tests {
         );
     }
 
+    #[tokio::test]
+    #[ignore]
+    async fn balancer_pool_fetcher_print() {
+        let transport = transport::create_env_test_transport();
+        let web3 = Web3::new(transport);
+        let chain_id = web3.eth().chain_id().await.unwrap().as_u64();
+        let contracts = BalancerContracts::new(&web3).await.unwrap();
+        let token_info_fetcher =
+            Arc::new(CachedTokenInfoFetcher::new(Box::new(TokenInfoFetcher {
+                web3: web3.clone(),
+            })));
+        let block_stream =
+            crate::current_block::current_block_stream(web3.clone(), Duration::from_secs(1000))
+                .await
+                .unwrap();
+        let deny_list = vec![H256(hex_literal::hex!(
+            "072f14b85add63488ddad88f855fda4a99d6ac9b000200000000000000000027"
+        ))];
+        // let deny_list = vec![];
```

Comment:
nit: probably leftover

Reply:
That was intentional so I could compare the output with and without deny list. Makes it easier for future testers so I'd leave it in.